index.html

```diff
@@ -34,6 +35,179 @@ <h1 class="header-text">I’m Hamma
+                    See Project
+                </button>
+            </div>
+            <div class="work-card work-card4">
+                <img src="img/works/work4.png" alt="work1">
+
+                <h2 class="work-title">Multi-Post
+                    Stories</h2>
+                <div class="work-info">
+                    <p class="first">CANOPY</p>
+                    <p class="second">• Back End Dev</p>
+                    <p class="second">• 2015</p>
+                </div>
+                <div class="word-desc">
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                </div>
+                <div class="work-lang">
+                    <ul>
+                        <li>
+                            <p class="lang-text">html</p>
+                        </li>
+                        <li>
+                            <p class="lang-text">css</p>
+                        </li>
+                        <li>
+                            <p class="lang-text">javascript</p>
+                        </li>
+                    </ul>
+                </div>
+                <button type="button" class="work-btn btn">
+                    See Project
+                </button>
+            </div>
+        </section>
+        <section class="about-me">
+            <h1 class="header-text">About
+                Myself</h1>
+            <p>
+                I’m a software developer! I can help you build a product , feature or website Look through some of my
+                work and experience! If you like what you see and have a project you need coded, don’t hestiate to
+                contact me.
+            </p>
+            <a href="#">LET’S CONNECT</a>
+            <ul class="social-links-about">
+                <li><a href="#"><img src="img/icons/Linkedin_icon.png" alt=""></a></li>
+                <li><a href="#"><img src="img/icons/Vector.png" alt=""></a></li>
+                <li><a href="#"><img src="img/icons/Vector-1.png" alt=""></a></li>
+                <li> <a href="#"><img src="img/icons/Vector-2.png" alt=""></a></li>
+            </ul>
+            <button type="button" class="about-btn btn">
+                    Get my resume
+                </button>
+            <div class="skills-container">
+                <ul class="skills-group">
+                    <li class="skill-head">
+                        <p>Languages</p>
+                        <i class="fa-solid fa-arrow-down"></i>
+                    </li>
+                    <li class="skill-item">
+                        <img src="./img/icons/js.png" alt="javascript">
+                        <p>javascript</p>
+                    </li>
+                    <li class="skill-item">
+                        <img src="./img/icons/html.png" alt="html">
+                        <p>HTML</p>
+                    </li>
+                    <li class="skill-item">
```

Comment:
- [ ] please kindly reduce the space between the social media icons to be the same as those of figma, the direction icons should be the same also.
your design 
![Screenshot (73)_LI](https://user-images.githubusercontent.com/65298048/170694045-17a20dbe-56da-464a-806c-5d787312ec19.jpg)
figma design 
![Screenshot (72)_LI](https://user-images.githubusercontent.com/65298048/170694152-dbdb5e57-f8bb-4abf-9734-e72321175c75.jpg)